.github/workflows/foreman_plugin_workflow_test.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: Test foreman_plugin.yml
+
+on:
+  pull_request:
```

Comment:
should this have `on: push` too? unsure

Reply:
and maybe limit to changes to `foreman_plugin.yml`?

Reply:
Yes, let's limit to `foreman_plugin.yml`. And doing so on push isn't all that relevant IMHO. It does build a cache,but is that really needed in this repo?

Reply:
the cache is not, no.

the other benefit is that you run the test on the merge-result instead of the PR, which *can* differ

Reply:
I rarely look at those results. Modtly because I turned off GHA email notifications